### DISCUSSION TITLE
Series watched

### DISCUSSION THREAD

**Discussion by pecbeck** [sbondCo/Watcharr#574] [Q&A]
Is there a way to mark , say, series 1 of a 3 part series watched, but leave series 2 and 3 as planned to watch

**Comment by IRHM** [comment]
Hi @pecbeck, if it is three parts as in three episodes, you can mark the second and third as planned. You can keep the show on watching or planned too.

**Reply by pecbeck** [reply]
I was thinking more about several seasons (or series in uk), where I can mark the first as watched, and show the 2nd and third as planned?

**Reply by IRHM** [reply]
@pecbeck oh i see, you should see a status option above the first episode, that'll let you set the season status to planned. Is that what you were looking for?

**Reply by pecbeck** [reply]
Sorry, not getting it yet.  So I have 3 seasons/series of outlaws. I've watched series one, and have set it to watched, and s2 and 3 to planned, but when I set filter to show planned, outlaws isn't there. Am I being dim again.

**Reply by IRHM** [reply]
@pecbeck Ah, the watchlist filter only filters the main show status. Is the show Outlaws set to Planned?

**Reply by pecbeck** [reply]
Okay, so I have watched season one, and want to set it to finished, but want want seasons 2 and 3 set to planned, then when I have watched season 2, I want to set it to finished, so now season 3  is planned

**Reply by IRHM** [reply]
@pecbeck sorry i'm a little confused, did it work?

**Reply by pecbeck** [reply]
Ok , think I understand it now. I mark season 1 as finished, and season 2,3 as planned. However, outlaws doesn't show up in planned filter.

**Reply by IRHM** [reply]
@pecbeck That sounds correct, assuming the outlaws show itself doesnt have a status of planned.
